.pipelines/cherrypick/CherryPick.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+name: Cherry Pick commits from PR
+
+trigger:
+- fasttrack/*
+- tvuong/fasttrack/*
+pr: none
+
+variables:
+  - name: pipelinesRepoFolder
+    value: 'CBL-Mariner-Pipelines'
+
+  - name: pipelinesRepoRoot
+    value: $(Agent.BuildDirectory)/$(pipelinesRepoFolder)
+
+resources:
+  repositories:
+    - repository: CBL-Mariner-Pipelines
+      type: git
+      name: mariner/CBL-Mariner-Pipelines
+
+jobs:
+  - job: 'CherryPick'
+    pool: mariner-core-x64-1es-ubuntu-22.04
+    timeoutInMinutes: 15
+    displayName: 'Cherry-pick commit'
+
+    steps:
+    - checkout: CBL-Mariner-Pipelines
+      path: $(pipelinesRepoFolder)
+      displayName: 'Checkout CBL-Mariner'
+
+    - template: spec-auto-patch-update/pipelines/CherryPick.yml@CBL-Mariner-Pipelines
+      parameters:
+        commitId: $(Build.SourceVersion)
+        sourceBranch: $(Build.SourceBranch)
+        repoRootPath: $(pipelinesRepoRoot)
```

Comment:
Is it possible to make this pipeline live in our ADO repositories entirely? This pipeline will require additional permissions and secrets access and there's no need to risk exposing it to malicious modifications from forked repos, which may slip through the cracks.

Reply:
I looked into that, but unfortunately resources trigger doesn't work for Github repos as of now: https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#triggers. 

> This pipeline will require additional permissions and secrets access

If I remember correctly, the default permissions won't allow forks to start this pipeline.

Reply:
Nevermind, looks like [this is not supported](https://learn.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/resources-repositories-repository?view=azure-pipelines#remarks). In that case, could you add a line in `.github/CODEOWNERS` to make any changes to this file require an approval from `@microsoft/cbl-mariner-admins`?

Reply:
And yes, we can (even must) disable fork builds, but I want to minimize a chance of a malicious PR getting approved and merged.

Reply:
I updated `.github/CODEOWNERS`. Since we don't have a `pipelines/` folder (without the dot), I updated that entry to `.pipelines/`